src/components/ProcessingPage/EditingPage/index.jsx

```diff
@@ -18,12 +21,20 @@ function useProgressStatus() {
     },
     // mockup시연의 경우 짧게 설정하고, 이후는 5-10초로 설정합니다.
     refetchInterval: 1000,
+    enabled: videoStatus,
```

Comment:
videoStatus가 Boolean 값으로 보이는데, 변수명이 명사라서 약간의 혼돈이 있을 것 같습니다!
`isVideoInProgress` 등과 같이 컨벤션을 맞춘 변수명이라면 더 가독성이 올라갈 것 같다는 의견입니다 :)

Reply:
맞네요 수진님! 해당 값은 `boolean`임으로 `boolean` 컨벤션을 따라 변수명 수정하겠습니다!
```suggestion
    enabled: isEnable,
```